src/scripts/wildBattle/Routes.ts

```diff
@@ -1,5 +1,7 @@
 /// <reference path="../../declarations/enums/Badges.d.ts"/>
 ///<reference path="RegionRoute.ts"/>
+///<reference path="../../declarations/enums/Badges.d.ts"/>
```

Comment:
Badge enums are referenced twice, although the scripts are (frighteningly) forgiving with references